paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass.cc

```diff
@@ -45,6 +45,7 @@ void LogCannotQuantizeOp(Node* op, const char* details = nullptr) {
          << " (type: " << op->Op()->Type() << ", id: " << op->id() << ").";
   if (details) msg_ss << " " << details;
   PrettyLogDetail(msg_ss.str().c_str());
+  op->Op()->SetAttr("mkldnn_data_type", std::string("float32"));
```

Comment:
Later on, could we refactor these log functions? It looks `VLOG(4)` is enough for these log functions?

Reply:
Does function below `void LogQuantizationDisabled(Node* op)`  maybe also need to set `"mkldnn_data_type", std::string("float32")` ?